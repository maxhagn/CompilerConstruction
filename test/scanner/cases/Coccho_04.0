end
return
goto
if
var
not
and

End
iF
end_return
goto_var
notAnd