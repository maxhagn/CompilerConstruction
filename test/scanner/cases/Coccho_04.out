end
return
goto
if
var
not
and
id End
id iF
id end_return
id goto_var
id notAnd